instrumentation/rocketmq-client-4.8/library/src/main/java/io/opentelemetry/instrumentation/rocketmq/RocketMqConsumerExperimentalAttributeExtractor.java

```diff
@@ -17,8 +17,8 @@
     implements AttributesExtractor<MessageExt, Void> {
   INSTANCE;
 
-  private static final AttributeKey<String> MESSAGING_ROCKETMQ_TAGS =
-      AttributeKey.stringKey("messaging.rocketmq.tags");
+  private static final AttributeKey<String> MESSAGING_ROCKETMQ_MESSAGE_TAG =
+      AttributeKey.stringKey("messaging.rocketmq.message_tag");
```

Comment:
it looks like there's a constant for this in the `SemanticAttributes` class that we could instead